README.md

```diff
@@ -0,0 +1,55 @@
+# Positioning and Floating Elements
+This project consisted of making a clone of a New York Times article page.The New York Times has been a leader in experimenting with the inclusion of multimedia content in traditional stories. Which creates all kinds of interesting position questions.
+Here's the original page http://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0
+
+![screenshot](./form-screenshot.png)
+
```

Comment:
## [REQUIRED ]- FIX - The README
Your README should match the [standards](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) that have been set in the pathwright. It is among the [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) for your repo to get approved.
##
- For some reason, most of your images are not being displayed properly. An example can be found in your README for instance.
![README](https://user-images.githubusercontent.com/22345961/78498826-5c23d880-7755-11ea-9604-be7b33af2f74.png)

I presume you meant this: 👇 ?
```suggestion
![screenshot](./app_screenshot.png)
```


This and many all of the many my colleague has highlighted in the previous changes as well. What they are seeing is exactly what I seeing on my end. They are working perfectly well in localhost but not in production.

Reply:
We also don't have a live link to the site in your README 
- You can use [githack](https://raw.githack.com/) to set up a live link and embed it in your README.

Reply:
## [REQUIRED] - FIX - The positioning of the More in Space and Astronomy section
The 'More in Space and Astronomy' section should be fixed to match the original version;
- This is what we currently have: 👇 
![position_clone](https://user-images.githubusercontent.com/22345961/78499886-17e80680-775c-11ea-84aa-fdd065d9c669.png)
 - THis is what we should be aiming to achieve: 👇 
![position](https://user-images.githubusercontent.com/22345961/78499908-39e18900-775c-11ea-8e6f-60e7e1fbdf8a.png)
Resize it.

Reply:
## [REQUIRED] - FIX - The footer
We should also fix the footer as well.
- This is what we have currently: 👇
![footer_clone](https://user-images.githubusercontent.com/22345961/78499986-9d6bb680-775c-11ea-810e-ef8d83e6fe71.png)
- We should aim to have it look something close to this: 👇 
![footer](https://user-images.githubusercontent.com/22345961/78499996-b2e0e080-775c-11ea-8b0f-69ea20322a00.png)

Reply:
## [OPTIONAL] - REPLACE - The emended image and video.
I applaud the good job done into building this project.
- We can always try and improve on some areas that might make our site even more appealing.
- Consider these changes optional as the general minimum requirements have already been met in these sections. 
##
Our embedded video: 👇 
![stanford_clone](https://user-images.githubusercontent.com/22345961/78500086-30a4ec00-775d-11ea-8fcf-cb16d022af1e.png)
We may change it to match the original: 👇 
![stanford_professor](https://user-images.githubusercontent.com/22345961/78500095-3dc1db00-775d-11ea-91c1-b8e2218c1718.png)
Our embedded image: 👇 
![bicept_clone](https://user-images.githubusercontent.com/22345961/78500104-49150680-775d-11ea-9d8e-2d27d95758d9.png)

We may change it to match the original: 👇 
![bicept](https://user-images.githubusercontent.com/22345961/78500114-58944f80-775d-11ea-97e4-ec114e1b65d6.png)